src/templates/admin/elements/forms/accessible_messages.html

```diff
@@ -0,0 +1,19 @@
+{% comment %}
+  Note that messages only display through this element if they are not acti
+{% endcomment %}
+{% for message in messages %}
```

Comment:
The comment above is incomplete. I don't think we should duplicate messages here. I know some of the account views use messages—convert those to non_field_errors so we don't have to duplicate message output.

Reply:
> The comment above is incomplete.

Thanks--I'll remove it.

Reply:
Appreciate this thinking on not duplicating messages. Some of these views operate without Django forms, so it would not be possible to rely on non-field errors for everything. I've opted to keep the messages in the body of the page and disable the toastr messages just on these pages, by putting the toastr script in a block that can be overwritten with nothing.